tests/tls/tls-subjectaltname/test.rules

```diff
@@ -0,0 +1,2 @@
+alert tls any any -> any any (msg:"Test tls.subjectaltname option CLIENT"; flow:to_client; tls.subjectaltname; content:"|77 77 77 2e 62 69 6e 67 2e 63 6f 6d|"; sid:1;)
```

Comment:
Can not the content be clear text instead of encoded ?

Reply:
I think it can be. But, I'll add clear text content tests too just to be sure. Thank you!